apps/adapt-essentials-asset-fields/README.md

```diff
@@ -1,31 +1,75 @@
 ![App icon](doc/app-icon.svg) [![Install to Contentful](https://www.ctfstatic.com/button/install-small.svg)](https://app.contentful.com/deeplink?link=apps&id=6RGLVmXYrGaIIRIda1HwAC)
 
-# Adapt Essentials: Asset fields 
+# Adapt Essentials: Asset fields
 
-An App for making it easier to change title, description and filename of assets in bulk. It supports in place editing for multiple locales.
+An App for making it easier to change title, description and filename of assets in bulk.
+
+It supports in place editing for multiple locales.
 
 ![Alt text](doc/app.png)
 
 ## Why?
 
-The common pattern for adding alt and description to the image is to upload the asset and reference it with dedicated wrapper content type. Which makes it a bit tedious for the editors.
+The common pattern for adding alt and description to the image is to upload the asset and reference
+it with dedicated wrapper content type. Which makes it a bit tedious for the editors.
 
-The lean solution is to use title and description fields of the asset itself for the alt and description image attributes. But you can't impose any requirements that way. So it is easy to forget to fill in the title and description fields.
+The lean solution is to use title and description fields of the asset itself for the alt and description image
+attributes.
+But you can't impose any requirements that way. So it is easy to forget to fill in the title and description fields.
 
 This app makes it easier to go through assets and edit it in bulk.
 
+## Bug reports
+
+If you've found any issues, please open an issue here:
+[https://github.com/adaptdk/adapt-essentials-asset-fields/issues](https://github.com/adaptdk/adapt-essentials-asset-fields/issues)
+
+## Feature requests
+
+If you think the application is missing any features, please open an issue here:
+[https://github.com/adaptdk/adapt-essentials-asset-fields/issues](https://github.com/adaptdk/adapt-essentials-asset-fields/issues)
+
+## [Privacy Policy](https://adaptagency.com/privacy-policy)
+
+<!--- App part -->
+
 ## How to use it?
 
-The app itself is really simple. Assets are listed with the title, description and status. You can edit the fields in place. The changes are saved automatically. You can also toggle additional fields or locale.
+You can find the app at https://app.contentful.com/spaces/<space_id>/apps/app_installations/adapt-essentials-bulk-asset-fields/
 
-![Alt text](doc/settings.png)
+### Access it through **Apps** menu
 
-## Bug reports
+Once app is installed it can be accessed through **Apps** menu.
```

Comment:
Now that you are adding an app configuration page, the app will no longer auto-install and users will actually have to click the 'Install' button on the configuration page, so it might be worth calling that out.

Reply:
Fix https://github.com/contentful/marketplace-partner-apps/pull/998/commits/9363d074abf718768c1a32a5c1cf15b07e64f835